### ISSUE TITLE
Update eraser to have higher contrast

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/scenery-phet#928]
Eraser location: https://github.com/phetsims/scenery-phet/blob/main/images/eraser.png

Seen in https://github.com/phetsims/membrane-transport/issues/223#issuecomment-3006586217 as well as https://github.com/phetsims/center-and-variability/issues/636 and Graphing Quadratics. The erase doesn't have much contrast with the button color, and to solve this @brettfiedler suggested suggested increasing the stroke, as well as making it black. 

Here is an updated asset that has a 0.75pt black stroke around the eraser, which should improve this for placement on yellow and white buttons, as well as when disabled. 

I've attached both a png and svg, in case you prefer one over the other! 

![Image](https://github.com/user-attachments/assets/bd5021fb-c828-4075-b5ac-97e335894b6c)
![Image](https://github.com/user-attachments/assets/8e85a2d9-e89e-4f53-9c16-7d357cf9bbab)

**Comment by samreid** [comment]
Updated in the commit, here is the before and after:

<img width="865" alt="Image" src="https://github.com/user-attachments/assets/3f769a2d-2912-45b0-84bd-cbb1f38ac29f" />

**Comment by amanda-phet** [comment]
Is that screenshot the disabled look of the button? 

@brettfiedler what do you think?

**Comment by amanda-phet** [comment]
I was able to look in context in the sim, and I think it looks pretty good! @brettfiedler if you're happy with it, please feel free to close!

**Comment by samreid** [comment]
Yes, the screenshot above is disabled, on startup, in membrane transport.

**Comment by brettfiedler** [comment]
This looks good in Membrane Transport. Big boost for the disabled state as well. thanks!
